src/Analysis/GWASV2/Shared/GWASCovariates/ContinuousCovariates.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './GWASCovariates.css';
+import Covariates from './Utils/Covariates';
+
+const ContinuousCovariates = ({
+  setMode,
+  sourceId,
+  // searchTerm,
+  selectedCovariate,
+  handleCovariateSubmit,
+}) => (
+  <React.Fragment>
+    <Covariates
+      selectedCovariate={selectedCovariate}
+      handleCovariateSelect={handleCovariateSubmit}
+      sourceId={sourceId}
+      // searchTerm={searchTerm}
+    />
+    <button type='button' style={{ marginLeft: 5 }} onClick={() => setMode('')}>Submit</button>
```

Comment:
the current code returns to the start of step2 when submit is clicked. Instead, it should go to step3 once a selection is submitted, given that we can have only 1 outcome phenotype.